javascript/index.js

```diff
@@ -14,10 +14,20 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
+  secUni.innerHTML = chronometer.getSeconds()
+
   // ... your code goes here
 }
 
 function printMinutes() {
+  //My idea is that I need to use double digit function and somehow get the first and second numbers. 
+  // I dont know how but for example, for minDec using [0] position of the 2 numbers
+  //and for the minUni the [1] position.  I dont know how!!
```

Comment:
yes! using bracket notations was the correct approach! you just needed to see how to access those values. you know that you have access to the chronometer object (see line 1) that holds all the attributes and methods you created on chronometer.js. So what you need to do is invoke those methods and use the results. for Example:
`chronometer.getSeconds()` will get you the current seconds as a number
`chronometer.twoDigitsNumber()` will transform any number you pass as a two digit string
so...
`chronometer.twoDigitsNumber(chronometer.getMinutes())` 
... will get you the current seconds as a string and you could apply bracket notation to it like you mentioned.